### PR TITLE
Provided RTC feature for nucleo-l073

### PR DESCRIPTION
**Pull request by In0ut**
it should be compatible with all nucleo-l0 boards, but tested only on nucleo-l073

### CODE REVIEW

File: boards/nucleo-l073/Makefile.features [hunk 1]
```diff
@@ -6,6 +6,7 @@ FEATURES_PROVIDED += periph_pwm
 FEATURES_PROVIDED += periph_spi
 FEATURES_PROVIDED += periph_timer
 FEATURES_PROVIDED += periph_uart
+FEATURES_PROVIDED += periph_rtc
```
Comment (vincent-d) [inline boards/nucleo-l073/Makefile.features]: Please sort them alphabetically (between `periph_pwm` and `periph_spi`)

**Comment by In0ut** [comment]
i think its ok now :)

File: boards/nucleo-l073/include/periph_conf.h [hunk 2]
```diff
@@ -182,7 +182,7 @@ static const spi_conf_t spi_config[] = {
  * @name    RTC configuration
  * @{
  */
-#define RTC_NUMOF           (0U)
+#define RTC_NUMOF           (1)
```
Comment (aabadie) [inline boards/nucleo-l073/include/periph_conf.h]: should be 1U

**Comment by aabadie** [comment]
I'll test tomorrow as I have a few L0 based boards.

**Comment by In0ut** [comment]
changed (1) to (1U)

**Comment by aabadie** [comment]
Tested tests/periph_rtc on Nucleo-L073 and Nucleo-L053 (with some adaption) and it works for both.

Can you also apply the changes to nucleo-l053 and nucleo32-L031 ?

The commit messages are also important and the history of your PR should be split in 4 commits:
* cpu/stm32-common: add RTC support to stm32l0
* boards/nucleo-l073: configure RTC
* boards/nucleo-l053: configure RTC
* boards/nucleo32-l031: configure RTC

**Comment by aabadie** [comment]
any problems with git ?

**Comment by In0ut** [comment]
HAHAHAHA

yes, but thats ok got back my commit

**Comment by In0ut** [comment]
the new PR is available there with the proper querry :

https://github.com/RIOT-OS/RIOT/pull/7131

2017-06-02 19:15 GMT+02:00 Alexandre Abadie <notifications@github.com>:

**Pull request closed without merging**